src/parser.js

```diff
@@ -77,56 +130,29 @@ export function containIncludeCommand(commands = []) {
  */
 export function parseVariablesFromLabel(kvMap, label) {
     const kv = Object.assign({}, kvMap);
-    const beginEx = "^.*";
-    const endEx = ".*$";
-    const sepEx = ",?";
-    const kvsepEx = "[:=]";
-    const spacesEx = "\\s*";
-    const quotesEx = "[\"']";
-
-    Object.keys(kv).forEach(key => {
-        let keyEx = "(" + key + ")";
-        let valEx = "([-\\w\\s]*)";
-        if (key === "marker") {
-            keyEx = "(import|include)";
-            valEx = "(([-\\w\\s]*,?)*)";
+
+    let match = "";
+    while ((match = expressionRegExp.exec(label))) {
+        let key = match[1];
+        if (["include", "import"].includes(key)) {
```

Comment:
Node.js v4 does not support Array#inclues 
http://node.green/#ES2016-features-Array-prototype-includes-Array-prototype-includes
We want to support Node.js LTS as possible.

Can you use `indexOf` or add ponyfill or polyfill?